### PR TITLE
Rewrite templates as yaml forms to improve clarity

### PR DESCRIPTION
**Pull request by denisonbarbosa**
With https://github.com/canonical/desktop-engineering/pull/2, we defined some yaml forms to be used as issue templates for reporting bugs and requesting features on GitHub.
This provides a better and more defined way for users to interact with us and provide us with the relevant information we need to solve the problem

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+name: Report an issue
+description: File an issue
+title: ""
+labels: ""
```
Comment (denisonbarbosa) [inline .github/ISSUE_TEMPLATE/bug_report.yml:4]: Should we prefix the title with something and/or add a "bug" label?

**Review by didrocks — CHANGES_REQUESTED** [review]
Good idea!

Can you modify https://github.com/canonical/desktop-engineering/blob/main/project-repo/templates/common/.github/ISSUE_TEMPLATE/bug_report.md too (and same for feature requests), and merge any additions that were made there here too? So that we really follow the same template and content?

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  - type: textarea
+    attributes:
+      label: "Non Ubuntu users: System information and logs"
+      description: |
+        For users of other distributions than ubuntu, provide details about the environment you experienced the issue in:
+      value: |
+        ### Environment
+        * aad-auth version: please run `aad-cli version`
+        * Distribution: (**NAME** in `/etc/os-release`)
+        * Distribution version: (**VERSION_ID** on `/etc/os-release`):
```
Comment (didrocks) [inline .github/ISSUE_TEMPLATE/bug_report.yml:69]: same here than adsys, tell to paste the logs?

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  - type: textarea
+    attributes:
+      label: "Non Ubuntu users: System information and logs"
+      description: |
+        For users of other distributions than ubuntu, provide details about the environment you experienced the issue in:
+      value: |
+        ### Environment
+        * aad-auth version: please run `aad-cli version`
+        * Distribution: (**NAME** in `/etc/os-release`)
+        * Distribution version: (**VERSION_ID** on `/etc/os-release`):
```
Comment (didrocks) [inline .github/ISSUE_TEMPLATE/feature_request.yml:65]: same here than adsys, tell to paste the logs?

**Review by didrocks — CHANGES_REQUESTED** [review]
I think one that one, the throubleshooting guide is fine. We just need to tell to paste the logs.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+        ### Application settings
+        Please redact/remove sensitive information:
+        ```raw
+        You can get the configuration file from /etc/aad.conf
+        ```
+  - type: textarea
+    attributes:
+      label: Relevant information
+      description: >
+        Please look at our [Troubleshooting guide](../#troubleshooting) and provide logs for the PAM and NSS modules.
```
[the remark this answers is not in the document]
Comment (denisonbarbosa) [reply .github/ISSUE_TEMPLATE/bug_report.yml:87]: Yeah, it works. I've already tested it on https://github.com/denisonbarbosa/testing-issue-template.

**Review by didrocks — APPROVED** [review]

**Comment by denisonbarbosa** [comment]
Since this has no impact at all on the tests, I'll merge it without the tests passing while I handle the CI fails.

**Pull request merged**